src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -881,6 +897,50 @@ public async Task TestFilterWithEntityNameAlias()
             await ExecuteAndValidateResult(_graphQLQueryName, gqlQuery, dbQuery);
         }
 
+        /// <summary>
+        /// MoonAdditionalAttributes (array inside moon object which is an array in container): "@item.name eq 'moonattr0'"
+        /// Earth(object in object): "@item.type eq 'earth0'"
+        /// AdditionalAttribute (array in container): "@item.type eq 'volcano0'"
```

Comment:
I don't understand what you're explaining in this comment. Explain relevance of the Policy text you're referencing.

Reply:
added.